samples/rabbitmq/simple/sample.md

```diff
@@ -10,21 +10,18 @@ related:
 ---
 
 
-## Code walk-through
+## Prerequisites
 
-This sample shows a basic usage of RabbitMQ as a transport for NServiceBus.
+Ensure you have a running instance of RabbitMQ.
 
 
-### Enable RabbitMQ
+## Code walk-through
 
-snippet:ConfigureRabbit
+This sample shows basic usage of RabbitMQ as a transport for NServiceBus. The application sends an empty message to itself, via the RabbitMQ broker, and writes to the console when the message is received.
 
 
-### Configuring RabbitMQ
+### Configuration
```

Comment:
It's debatable, but it was previously part of "Code walk-through" and is also under there in https://docs.particular.net/samples/sqltransport/simple/ and https://docs.particular.net/samples/azure/azure-service-bus/